rdbms/src/main/java/io/spine/server/storage/jdbc/StandardMapping.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.storage.jdbc;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.spine.type.TypeName;
+
+import java.sql.DatabaseMetaData;
+import java.sql.SQLException;
+
+import static io.spine.server.storage.jdbc.BaseMapping.baseBuilder;
+import static io.spine.server.storage.jdbc.Type.BYTE_ARRAY;
+
+/**
+ * Standard {@linkplain TypeMapping type mappings} for different databases.
+ *
+ * @author Dmytro Grankin
+ */
+public enum StandardMapping implements TypeMapping {
+
+    MYSQL_5("MySQL", 5, baseBuilder()),
```

Comment:
We have discussed this previously. Please specify the version properly. Five isn't specific enough.

Reply:
Done.